Gemfile

```diff
@@ -1,3 +1,5 @@
-source "https://rubygems.org"
-
-gem 'github-pages'
```

Comment:
Before this gets merged, I don't know what impact removing this will have on the GitHub site.  Everything else shouldn't have an impact.